policyengine_us/parameters/gov/states/md/tax/income/deductions/standard/min.yaml

```diff
@@ -16,17 +18,57 @@ metadata:
       href: https://www.marylandtaxes.gov/new-tax-year-update.php#:~:text=Standard%20Deduction%20%2D%20The%20tax%20year,spouse%2C%20and%20taxpayers%20filing%20jointly.
 
 JOINT:
-  2018-01-01: 3_000
-  2021-01-01: 3_100
+  values:
+    2018-01-01: 3_000
+    2021-01-01: 3_100
+    2023-01-01: 3_450
```

Comment:
didn't change in 2022? please add a 2022 tax form ref

Reply:
Missed that, thank you, fixed